setup.cfg

```diff
@@ -125,6 +125,8 @@ warn_untyped_fields = True
 # D213 Multi-line docstring summary should start at the second line (reason: pep257 default)
 # D215 Section underline is over-indented (reason: pep257 default)
 ignore = D105, D107, D203, D213, D215
+# Allow missing docstrings in methods that are overridden
+ignore_decorators = override
```

Comment:
Oooh nice. I think I'll include this in the ruff config in starbase. https://docs.astral.sh/ruff/settings/#pydocstyle-ignore-decorators